### DISCUSSION TITLE
One file, one encapsulated item convention

### DISCUSSION THREAD

**Discussion by Falesz** [Buduf/PTI-Ear-Trainer#8] [Ideas]
Hey all,
I'm wondering how strictly we should follow the abovementioned convention. I see in the current model enums are embedded into the model class. I'd like to organize these into their own separate source file. If you have no objection, I'll get to this shortly.

**Comment by Andre2220** [comment]
It's okay for me, I don't have any problem with it. I've just started and created it there, but you can move it to a separate source file if you'd like.
